modules/user-access/pages/query-privilege-migration.adoc

```diff
@@ -0,0 +1,136 @@
+= Query Privilege Migration
+
+This page explains the procedures for the query privilege migration with an eligible version to upgrade from to 4.1.0.
+
+To see user management tasks under the xref:access-control-model.adoc#_access_control_lists[Access Control List (ACL)] model, see xref:acl-management.adoc[].
+
+To see Role-based Access Control(RBAC) Object-Based privileges, see xref:rbac-row-policy/rbac-row-policy.adoc#_object_based_privileges[].
+
+== Migrate ACL Privileges
+Since 4.1.0, Access Control Lists (ACLs) should be deprecated.
+
+Query ACL privileges should be migrated automatically during upgrade. 
+
+[#_migrate_query_acl_owner]
```

Comment:
Is it true that to initiate this migration, the user must have WRITE_USER and OWNERSHIP privileges on the relevant queries?

If yes, explicitly mention it.

Reply:
It is not necessary. It's an internal process when GSQL is starting after the upgrade.